rascaline/src/calculators/lode/spherical_expansion.rs

```diff
@@ -61,6 +63,48 @@ impl LodeSphericalExpansion {
             spherical_harmonics,
         });
     }
+
+    /// Compute the trigonometric functions for LODE coefficients
+    fn compute_structure_factors(&mut self, positions: &[Vector3D], k_vectors: &[KVector]) -> (Array3::<f64>, Array3::<f64>) {
```

Comment:
Also, this does not need the `self` parameter, so it might be better as a standalone function

```suggestion
    fn compute_structure_factors(positions: &[Vector3D], k_vectors: &[KVector]) -> (Array3::<f64>, Array3::<f64>) {
```